R/ee_roll_stat.R

```diff
@@ -0,0 +1,61 @@
+
```

Comment:
Why are something things in the R directory, and others in exploration

Reply:
- `R/` holds finished work that is sourced for analysis
- `exploration/` I am basically using as a staging ground to save work as I develop it into R targets, maybe there is a better place to do this? Almost everything in exploration will be deleted or completely redone so I don't even think it is worth reviewing?